### ISSUE TITLE
Not Working Anymore. 

### ISSUE THREAD

**Issue by LordYesLord** [juneyourtech/GM_YT#3]
First of all this is my first time using this website, so excuse me if I do something wrong.
Secondly I want to report a problem, until today 12/09.2017 this script worked just fine, now it doesn't load anymore.
I'm using firefox, greasemonkey and youtube flash video player 54 and disabling the webm settings in about config. Until today that worked just fine, however now I get "your browser does not currently recognize any of the video formats available. Click here for html5 questions."
Can this be fixed or youtube really wants to shove html5 down our throats?
Thank you.

**Comment by mirh** [comment]
It seems they shut off even embedded flash videos... 
We are left wondering if other handhold exist

**Comment by juneyourtech** [comment]
Yeah, well, I'm getting the same symptoms. Stand-alone embeds do not show Flash either.

**Comment by juneyourtech** [comment]
@LordYesLord The upated Flash userscript was [here](https://github.com/juneyourtech/GM_YT_Flash), but it's maybe good to know, that the older version worked, too.

As stated above, it cannot be fixed, unless someone more clever than I provides the necessary code to get Flash back in such a way, that it's publicly available from YouTube itself. Some other (deeper) methods might stretch the limits of permissibility.

The best I can say now, is, that it was good as long as it lasted.

**Comment by LordYesLord** [comment]
@juneyourtech My apologizes for posting in the wrong place, and thank you for the reply. So for now no fix, bummer.
Well like you said at least it was good while it lasted.

**Comment by juneyourtech** [comment]
@LordYesLord That's all right. Just tonight I discovered, that the link to the older release was [passed](https://addons.mozilla.org/en-US/firefox/addon/youtube-flash-player/reviews/) [around](https://addons.mozilla.org/en-US/firefox/addon/youtube-flash-video-player/reviews/) in the reviews/comments to two of YouTube/Flash-enabling extensions at the Mozilla Add-ons site. I didn't know, that the userscript had gained any traction beyond just a handful of people. And it all lasted for a little more than one month.
